### ISSUE TITLE
Error with empty cards

### ISSUE THREAD

**Issue by TheChilliPL** [Lattyware/massivedecks#166]
When playing the game with code `GWOD` (and a password) that still didn't end, I tried to pass an empty card with custom text. At first, I couldn't send it at all. Then, after a reload, I sent it, and it got sent but without the text I put in and it threw an error that the server didn't know how to interpret a request or something.
I don't have the full error, but I presume you have the logs of the game somewhere

**Comment by Lattyware** [comment]
Yeah, that sounds like a bug. I don't remember exactly, but I don't think there should be anything stopping you playing an empty card. I assume we are doing something wrong along the chain where we equate emptiness with no card, causing some issue. In general the code is defensive against that kind of thing, but nothing is impossible. I'll take a look when I get a chance.
